packages/client/src/components/AutoSuggestList.tsx

```diff
@@ -38,8 +43,8 @@ const StyledIcon = styled(Icon)`
 const SuggestList: React.FC<{
   activeIndex: number;
   className?: string;
-  onSelectOption: Function;
-  options: [{ id: number; value: string }];
+  onSelectOption: (arg0: Option) => void;
+  options: [Option];
```

Comment:
Bit off topic...
Not entirely sure but I thought `[Option]` is a list with 1 option. and `Option[]` is a list of Options. So can be more than one.

Reply:
I think you're correct. Made it `Option[]` instead of `[Option]`. Both seem to work though